### PR TITLE
Add a comment on taking over stale PRs

### LINKED ISSUE #939: Need to document policy for incomplete pull requests
- [ ] Ask before taking over
- [ ] What is an appropriate time to wait before swooping in?

### PR DESCRIPTION
**Pull request by betatim**
Fixes #939

- [x] Is it mergeable?
- [x] `make format diff_pylint_report cppcheck doc pydocstyle` Is it well
  formatted?
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by betatim** [comment]
Running `make doc` locally pointed out:
```
..../dib/khmer/doc/dev/hackathon.rst:: WARNING: document isn't included in any toctree
..../dib/khmer/doc/dev/python-api.rst:: WARNING: document isn't included in any toctree
```
Is this on purpose or oversight?

**Comment by standage** [comment]
I'm pretty sure this is on purpose. We want to keep a record of the hackathon stuff, but it no longer needs to be rendered on the lab. The Python API docs are pretty nascent I think.

**Comment by betatim** [comment]
Ready for review! @luizirber, @camillescott, @standage , or @ctb

File: doc/dev/for-khmer-developers.rst [hunk 1]
```diff
@@ -50,15 +50,16 @@ First, never merge something unless it's been through a review!  This
...
 
-Second, need to force another continuous integration run? Put "test
-this please" in a comment.  This is used to ask our continuous
-integration system to run on someone else's pull request -- by
-default, it only runs on commits from people who have write privileges
-to khmer, so you may need to do this if you're reviewing someone else's
-pull request.
-
-Third, we ask that all contributors set up standing Pull Requests
+Second, we ask that all contributors set up standing Pull Requests
```
Comment (standage) [inline doc/dev/for-khmer-developers.rst]: Currently this spells out Pull Request several times, and then uses the abbreviation PR several times. Perhaps we should follow the convention of introducing the abbreviation after the first occurrence in a section, and then using the abbreviation for all subsequent occurrences.

**Pull request merged**